src/views/DaoInfo/components/StatsRibbon/StatsRibbon.tsx

```diff
@@ -100,7 +100,9 @@ const StatsRibbon: React.FC<StatsRibbonProps> = ({ blockHeight }) => {
           <Chip label={"Circulating INK: " + circulatingSupply} color="primary" variant="outlined" />
         </Tooltip>
         <Chip label={"Ecosystem TVL: $" + ecosystemTvl} color="primary" variant="outlined" />
-        <Chip label={"MarketCap: $" + currentMarketcap} color="primary" variant="outlined" />
+        <Tooltip title={"Fully Diluted: $" + new BigNumber(tokenPrices[INK] * 88 * 1000000).toFormat(0)}>
```

Comment:
wouldn't it be better to handle the bigNumber math with `new BigNumber(tokenPrices[INK]).times(88000000)`

reason being, it's using bignumbers parsing properly and should not lead to an overflow or weird math issue

Reply:
fixed